cognite/client/data_classes/_base.py

```diff
@@ -136,7 +136,10 @@ def _load(
         raise TypeError(f"Resource must be json str or dict, not {type(resource)}")
 
     def to_pandas(
-        self, expand: Sequence[str] = ("metadata",), ignore: Optional[List[str]] = None, camel_case: bool = False
+        self,
+        expand: Sequence[str] = ("metadata",),
+        ignore: Optional[List[str]] = None,
+        camel_case: bool = False,
```

Comment:
why did you add the extra comma here, you did no changes to this function? 
```suggestion
        camel_case: bool = False
```

Reply:
I accidentally put my code in `CogniteResource` instead of `CogniteResourceList`, that is why. 😅